packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -276,3 +280,24 @@ export async function compareHash(plaintext: string, hashed: string): Promise<bo
 		throw new Error(error);
 	}
 }
+
+// return the difference between two arrays
+export function rightDiff<T1, T2>(
```

Comment:
this feels like it doesn't belong here. maybe we can either make it a helper in `core` or either find a node module that does this, or publish this as our own module to npm.